bin/plotting/pycbc_page_foundmissed

```diff
@@ -59,6 +75,10 @@ elif args.far_type == 'exclusive':
     ifar_found = f['found_after_vetoes/ifar_exc'][:]
     far_title = 'Exclusive'
 
+if args.limit_ifar:
+    ifar_above_thresh = ifar_found > args.limit_ifar
```

Comment:
could you replace these two lines with a single call to eg numpy.minimum (element-wise minimum)?